app/models/application.rb

```diff
@@ -0,0 +1,41 @@
+class Application < ApplicationRecord
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zipcode, presence: true, numericality: true
+  validates :description, presence: true
+  has_many :application_pets
+
+
+  def self.application(id)
+    Application.find(id)
```

Comment:
This is a bit of encapsulation overkill IMO -- you can just call .find from the controller and that's not a violation of MVC!